lib/ts/recipe/session/index.ts

```diff
@@ -40,7 +40,7 @@ export default class SessionWrapper {
         sessionData: any = {},
         userContext: any = {}
     ) {
-        const defaultClaims = Recipe.getInstanceOrThrowError().getDefaultClaims();
+        const defaultClaims = Recipe.getClaimsAddedByOtherRecipes();
```

Comment:
Need to change var name as well. I know it's just scoped to this function, but still